Central-Server/API/database.py

```diff
@@ -0,0 +1,24 @@
+import psycopg2
+
+import os
+def get_db_secret():
+    with open(os.getenv("DB_PASSWORD_FILE")) as f:
+        return(str(f.read()))
+
+db_secret = get_db_secret().strip()
+
+def get_db_name():
+    val = os.getenv("DB_NAME")
+    if val == None:

```

Comment:
allegedly `is None` is better(?) take that as u will

Reply:
https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none

So it is, I'll bring it up